cmd/uncloud/deploy.go

```diff
@@ -49,9 +53,22 @@ func NewDeployCommand() *cobra.Command {
 	return cmd
 }
 
+// projectOpts returns the project options for the Compose file(s).
+func projectOpts(opts deployOptions) []composecli.ProjectOptionsFn {
+	projectOpts := []composecli.ProjectOptionsFn{}
+
+	if len(opts.profiles) > 0 {
+		projectOpts = append(projectOpts, composecli.WithProfiles(opts.profiles))
```

Comment:
Did you consider using `WithDefaultProfiles` that falls back to profiles specified via the `COMPOSE_PROFILES` environment variable to be more compatible with the default `docker compose` behaviour?

https://github.com/docker/compose/blob/5bb46035cf12e5b2fc2f3b09522a392b7b753120/cmd/compose/compose.go#L418-L419

Reply:
Nice one, didn't see that. Thanks.